src/Makevars.win

```diff
@@ -1,17 +1,12 @@
-# -*- mode: Makefile -*-
-#
-# Copyright 2005 - 2006 Dominick Samperi
-# Copyright 2005        Uwe Ligges
-# Copyright 2008 - 2015 Dirk Eddelbuettel
-# Copyright 2011        Uwe Ligges, Brian Ripley, and Josh Ulrich
-

```

Comment:
It's a little unusual to remove old attribution so I'll probably put this back in.